### PR TITLE
Add Google, Apollo, CSE, Kingston, Mosaic, Sodexo, Indico sponsor logos

### PR DESCRIPTION
**Pull request by RobertWSaunders**
## Proposed Change

Adds a bunch of new sponsor logos!

### How did you do this?

Add the new logos to the Sponsor page!

### Why are you choosing this approach?

Make our sponsors happy!

### Any open questions you want to ask code reviewers?

Nope!

### List of changes:

  - Add Google 
  - Add Apollo
  - Add CSE
  - Add Kingston
  - Add Sodexo
  - Add Indico
  - Update Mosaic logo

## Pull Request Checklist:

  - [x] My submission passes all tests.
  - [x] I have lint my code locally prior to submission.
  - [x] I have written new unit tests for my core changes (where applicable).
  - [x] I have written browser integration tests for my core changes (where application).
  - [x] I have referenced all useful information (issues, tasks, etc) in the pull request.

### CODE REVIEW

**Comment by RobertWSaunders** [comment]
Deploy preview for *qhacks-website* ready!

Built with commit fb2c6f84f20de538f99b28eabef7e260c9a3446a

https://deploy-preview-245--qhacks-website.netlify.com

File: src/assets/img/sponsors/indico.svg [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joeytepp) [inline src/assets/img/sponsors/indico.svg]: It would appear this is 4 spaces 🔍

File: src/assets/img/sponsors/kingston.svg [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joeytepp) [inline src/assets/img/sponsors/kingston.svg]: This as well 🕵️

File: src/assets/img/sponsors/sodexo.svg [hunk 3]
```diff
@@ -0,0 +1,177 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (joeytepp) [inline src/assets/img/sponsors/sodexo.svg]: And this 🔬

File: src/components/Sponsors.js [hunk 4]
```diff
@@ -595,28 +620,90 @@ const Sponsors = () => (
         <div
           css={{
             display: "grid",
-            gridTemplateColumns: "1fr",
-            marginLeft: "auto",
-            marginRight: "auto",
-            width: "100%"
+            gridTemplateColumns: "1fr 1fr 1fr 1fr",
```
Comment (joeytepp) [inline src/components/Sponsors.js]: Nice job with the alignment, looks great 👍

**Pull request merged**